daemon/gdbus-util.c

```diff
@@ -141,3 +141,18 @@ gdbus_put_system_connection (void)
 {
   g_clear_object (&g_dbus_sys_conn);
 }
+
+/**
+ * @brief Common function to initailize the DBus module.
```

Comment:
minor typo `initailize` here and in gdbus-util.h

Reply:
updated